views/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "new-theme",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
+    "watch": "webpack --progress --colors --debug --display-chunks --watch",
+    "start-dev-server": "webpack-dev-server --config webpack.config.js",
+    "vue-test": "karma start js/app/test/unit/karma.conf.js"
+  },
+  "author": "PrestaShop",
+  "license": "OSL-3.0",
+  "devDependencies": {
+    "babel-core": "^6.25.0",
+    "babel-loader": "^6.2.1",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-stage-2": "^6.22.0",
+    "bootstrap-slider": "^6.0.16",
+    "bootstrap-tokenfield": "^0.12.0",
+    "bourbon": "^4.2.6",
+    "chai": "^3.5.0",
+    "chai-jquery": "^2.0.0",
+    "compass": "^0.1.1",
+    "css-loader": "^0.27.3",
+    "dropzone": "^5.0.0",
+    "eonasdan-bootstrap-datetimepicker": "^4.17.47",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb-base": "^11.3.1",
+    "eslint-import-resolver-alias": "^1.0.3",
+    "eslint-import-resolver-webpack": "^0.8.3",
+    "eslint-plugin-html": "^2.0.3",
+    "eslint-plugin-import": "^2.7.0",
+    "exports-loader": "^0.6.3",
+    "expose-loader": "^0.7.3",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "file-loader": "^0.10.1",
+    "imports-loader": "^0.6.5",
+    "jets": "0.11.0",
+    "jquery-ui-dist": "^1.12.1",
+    "jwerty": "^0.3.2",
+    "karma": "^1.7.0",
+    "karma-babel-preprocessor": "^6.0.1",
+    "karma-chai": "^0.1.0",
+    "karma-chai-jquery": "^1.0.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.1",
+    "karma-jquery": "^0.2.2",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-phantomjs-shim": "^1.4.0",
+    "karma-sinon-chai": "^1.3.1",
+    "karma-spec-reporter": "0.0.31",
+    "karma-webpack": "^2.0.4",
+    "loader-utils": "^1.0.3",
+    "lodash": "^4.17.4",
+    "magnific-popup": "^1.1.0",
+    "mobile-detect": "^1.3.5",
+    "mocha": "^3.4.1",
+    "moment": "^2.19.1",
+    "node-sass": "^4.5.0",
+    "path": "^0.12.7",
+    "postcss-cssnext": "^2.10.0",
+    "postcss-loader": "^1.3.3",
+    "prestakit": "^1.0.2",
+    "sass-loader": "^6.0.6",
+    "sinon": "^2.4.1",
+    "sinon-chai": "^2.12.0",
+    "sprintf-js": "^1.0.3",
+    "style-loader": "^0.14.0",
+    "tether": "^1.3.2",
+    "typeahead.js": "^0.11.1",
+    "uglify-save-license": "^0.4.1",
+    "url-polyfill": "^1.0.11",
+    "vue": "^2.4.2",
```

Comment:
not sure you need vue and many other packages here :/

Reply:
nope not at all, this was copied from the core It still needs some cleaning